### PR TITLE
Support initiative

### PR DESCRIPTION
**Pull request by KatiRG**
Added link for support button + one test.

### CODE REVIEW

File: tests/acceptance/supporting-an-initiative-test.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    Ember.run(App, 'destroy');
+  }
+});
+
+test('Successfullyyy', function(){
+  visit('/initiatives/fixture-0').then(function() {
+   	click( $("a:contains('Support this initiative')") ).then(function() {
+      expect(currentURL()).to.equal('/initiatives/fixture-0');
+      expect(find('.support').text()).to.equal('Remove Support for this initiative');
+      //expect( $("a:contains('I support this')") ).to.equal( $("a:contains('I support this')") );
```
Comment (oliverbarnes) [inline tests/acceptance/supporting-an-initiative-test.js]: always keep comments out of commits and pull requests

File: tests/acceptance/supporting-an-initiative-test.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    App = startApp();
+  },
+  teardown: function() {
+    Ember.run(App, 'destroy');
+  }
+});
+
+test('Successfullyyy', function(){
+  visit('/initiatives/fixture-0').then(function() {
+   	click( $("a:contains('Support this initiative')") ).then(function() {
```
Comment (oliverbarnes) [inline tests/acceptance/supporting-an-initiative-test.js]: fix indentations

File: app/routes/initiatives/show.js [hunk 3]
```diff
@@ -3,7 +3,22 @@ import Ember from 'ember';
 var InitiativesShowRoute = Ember.Route.extend({
   model: function(params) {
     return this.store.find('initiative', params.initiative_id);
+  },
+
+  actions: {
+    supportIt: function() {
+      console.log('supported!');
```
Comment (oliverbarnes) [inline app/routes/initiatives/show.js]: remove console.log calls from commits

File: tests/acceptance/supporting-an-initiative-test.js (hunk 1 above)
Comment (oliverbarnes) [inline tests/acceptance/supporting-an-initiative-test.js]: Add a test for the reverse toggle (clicking 'Remove support'), and possibly another test for the isSupported property on the model (cookie points)

File: tests/acceptance/supporting-an-initiative-test.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+    Ember.run(App, 'destroy');
+  }
+});
+
+test('Successfully', function(){
+  visit('/initiatives/fixture-0').then(function() {
+    click( $("a:contains('Support this initiative')") ).then(function() {
+      expect(currentURL()).to.equal('/initiatives/fixture-0');
+      expect(find('.support').text()).to.equal('Remove support for this initiative');
+      click( $("a:contains('Remove support for this initiative')") ).then(function() {
```
Comment (KatiRG) [inline tests/acceptance/supporting-an-initiative-test.js:22]: should this second click be within the first click or should it go outside (but still remain in visit)? Both ways work.

**Pull request closed without merging**